test-cargo-miri/tests/test.rs

```diff
@@ -49,14 +49,15 @@ fn cargo_env() {
 }
 
 #[test]
-#[should_panic(expected="Explicit panic")]
-fn do_panic() { // In large, friendly letters :)
+#[should_panic(expected = "Explicit panic")]
+fn do_panic() {
+    // In large, friendly letters :)
```

Comment:
```suggestion
fn do_panic() {
```
What rustfmt does here makes no sense (the comment comments on the fn signature, not the body). It's also not a critical comment...